Scripts/Services/NaturalLanguageUnderstanding/V1/Model/AnalysisResults.cs

```diff
@@ -44,7 +44,7 @@ public class AnalysisResults
         /// API usage information for the request.
         /// </summary>
         [JsonProperty("usage", NullValueHandling = NullValueHandling.Ignore)]
-        public Usage Usage { get; set; }
+        public AnalysisResultsUsage Usage { get; set; }
```

Comment:
Why are we changing from `Usage` here? The new change means `AnalysisResults` has `AnalysisResultsUsage` in there. Using `AnalysisResults` again sounds repetitive. Is this change coming from `oas3`?

Reply:
Yes this is coming from the generator. I'm not sure how it synthesized the name. We can look into this if you think it's a big deal though

Reply:
Im trying to find where this gets transformed in the generator - Let me check with @padamstx tomorrow.

Reply:
We can resolve this conversation based on our talks offline!